app/kuma-cp/cmd/run.go

```diff
@@ -53,8 +53,6 @@ func newRunCmdWithOpts(opts kuma_cmd.RunCmdOpts) *cobra.Command {
 				return err
 			}
 
-			kuma_cp.PrintDeprecations(&cfg, cmd.OutOrStdout())
```

Comment:
I don't think we should delete the actual calls to the `kuma_cp.PrintDeprecations`. We may need this mechanism in the future. You can simply make the `deprecations` list empty in `pkg/config/app/kuma-cp/deprecate.go`:
```go
var deprecations = []config.Deprecation{}
```

Reply:
addressed